SystemInformer/prpghndl.c

```diff
@@ -110,13 +110,13 @@ VOID PhpInitializeHandleMenu(
 
     level = KsiLevel();
 
-    if (level >= KphLevelMed)
-    {
-        if (item = PhFindEMenuItem(Menu, 0, NULL, ID_HANDLE_PROTECTED))
-            PhDestroyEMenuItem(item);
-        if (item = PhFindEMenuItem(Menu, 0, NULL, ID_HANDLE_INHERIT))
-            PhDestroyEMenuItem(item);
-    }
+    //if (level >= KphLevelMed)
```

Comment:
Looking back at history this check should have been `if (level < KphLevelMed)` instead of `>=`, oops. Nice find!

I think we should just deleted the commented out code entirely.

If we present users with the options and they use them with the driver disabled, a message will inform the user to enable the driver. An alternative would be to disable the menu item `PH_EMENU_DISABLED` when `if (level < KphLevelMed)`, but I think maybe the best experience is to just leave those options there and enabled.

Reply:
Ok, I removed commented lines entirely

Reply:
Thanks for doing that. And yep that is the intended required access for this path 👍.